WordPress/Classes/ViewRelated/Gutenberg/Layout Picker/GutenbergLayoutPickerViewController.swift

```diff
@@ -317,23 +322,24 @@ extension GutenbergLayoutPickerViewController: UITableViewDelegate {
     }
 
     private func containsSelectedLayout(_ layout: GutenbergSelectedLayout, atIndexPath indexPath: IndexPath) -> Bool {
-        let rowSection = sections[indexPath.row]
+        let rowSection = (filteredSections ?? sections)[indexPath.row]
```

Comment:
I thought about creating a getter for `(filteredSections ?? sections)` but I felt like that was clouding when I was explicitly trying to reference `filteredSections` or `sections` vs when I wanted the one that was appropriate for the current display mode